.github/workflows/license.yml

```
@@ -0,0 +1,14 @@
+name: License
+on:
+  push:
+    branches: [master]
+jobs:
+  licenses:
+    runs-on: self-hosted
```

afair, we can't use `self-hosted` runners on public repos.
```suggestion
    runs-on: ubuntu-latest
```